src/Controls/samples/Controls.Sample/Maui.Controls.Sample-net6.csproj

```diff
@@ -43,21 +43,20 @@
     <MauiSplashScreen Include="Resources\Images\dotnet_bot.svg" Color="#FFFFFF" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Content Remove="wwwroot\css\app.css" />
-    <Content Remove="wwwroot\index.html" />
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="wwwroot\css\app.css" />
-    <EmbeddedResource Include="wwwroot\index.html" />
-  </ItemGroup>
-
   <ItemGroup>
     <Content Remove="**\*.razor" />
   </ItemGroup>
   <ItemGroup>
     <RazorComponent Include="**\*.razor" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Update="wwwroot\css\app.css">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Update="wwwroot\index.html">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
```

Comment:
Should these be here? Doesn't the SDK do this automatically for files in `wwwroot`?